.gitignore

```diff
@@ -0,0 +1 @@
+__pycache__
```

Comment:
The pycache files are still present in the PR. you will have to do a git reset --soft followed by a git commit to remove them. Please always have a look at the diff before pushing and https://github.com/jajodiaraghav/Snake/pull/29/files after pushing.